packages/app/src/cli/services/local-storage.ts

```diff
@@ -17,13 +18,27 @@ export interface AppLocalStorageSchema {
   [key: string]: CachedAppInfo
 }
 
-let _instance: LocalStorage<AppLocalStorageSchema> | undefined
+export interface CurrentConfigLocalStorageSchema {
+  [key: string]: string
+}
+
+let _appLocalStorageInstance: LocalStorage<AppLocalStorageSchema> | undefined
+let _currentConfigLocalStorageInstance: LocalStorage<CurrentConfigLocalStorageSchema> | undefined
 
 function appLocalStorage() {
-  if (!_instance) {
-    _instance = new LocalStorage<AppLocalStorageSchema>({projectName: 'shopify-cli-app'})
+  if (!_appLocalStorageInstance) {
+    _appLocalStorageInstance = new LocalStorage<AppLocalStorageSchema>({projectName: 'shopify-cli-app'})
+  }
+  return _appLocalStorageInstance
+}
+
+function currentConfigLocalStorage() {
+  if (!_currentConfigLocalStorageInstance) {
+    _currentConfigLocalStorageInstance = new LocalStorage<CurrentConfigLocalStorageSchema>({
+      projectName: 'shopify-cli-app',
+    })
```

Comment:
would these two storages be stored in the same json?

Reply:
We actually had the question of how `projectName` was used. If it's to determine the name of the file, I'll change it, so that keys don't collide between the two stores.